src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html

```diff
@@ -142,6 +143,27 @@
       {{name}}: ${{price}}
     </template>
   </amp-list>
+
+  <!-- E-Commerce websites tipically use carousel layout to display personalized content.
```

Comment:
suggestion: 'It's common for e-commerce websites to display personalized content or recommendations in a carousel.'

Reply:
done